composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "yajra/laravel-datatables-oracle",
+    "name": "carvinglabs/laravel-datatables-oracle",
```

Comment:
Changing the name might break the installations. Will implement this on a new PR with credits to you. Thanks!